src/components/20-molecules/dropdown/index.js

```diff
@@ -156,7 +157,6 @@ class AXADropdown extends NoShadowDOM {
     // to define controlledness
     applyDefaults(this);
     // bound event handlers (so scope and de-registration work as expected)
-    this.handleWindowKeyDown = this.handleWindowKeyDown.bind(this);
```

Comment:
@danyball why removing this? as far as i know this is used to close the dropdown when you  click "away"

Reply:
Lit html is registering those handlers with `@keydown`. Everything is working as before: https://axa-ch.github.io/plib-feature/bugfix/1863_dropdown_keys/?path=/story/components-dropdown--story